juniper_official/vpn/l2vpn/check-l2circuit-pw-state.rule

```diff
@@ -75,22 +97,94 @@
             rule-properties {
                 version 1;
                 contributor juniper;
-                supported-healthbot-version 4.0.0;
+                supported-healthbot-version 4.3.0;
                 supported-devices {
+                    sensors isis-oc;
```

Comment:
Change the sensor name to actual "l2ckt"

Reply:
updated with l2ckt